src/_pytest/capture.py

```diff
@@ -62,8 +62,9 @@ def silence_logging_at_shutdown():
     # finally trigger conftest loading but while capturing (issue93)
     capman.start_global_capturing()
     outcome = yield
-    out, err = capman.suspend_global_capture()
+    capman.suspend_global_capture()
     if outcome.excinfo is not None:
+        out, err = capman.snap_global_capture()
```

Comment:
Since we are refactoring things, how do you feel about renaming this to `read_global_capture`?

Reply:
It is just that `snap_global_capture` does `self._global_capturing.readouterr()`, so I think renaming it here makes sense. Snap is then used just for the "Capture" classes. 😁

Reply:
Makes sense, will do it